aws/requests.yaml

```
@@ -2,7 +2,7 @@ releases:
 - name: ">= 20.0.0"
   requests:
   - name: observability-bundle
-    version: ">= 1.1.0"
+    version: ">= 1.2.2"
```

New release out! :grin: 
```suggestion
    version: ">= 1.2.3"
```